src/chapters/classes/index.rst

```diff
@@ -0,0 +1,11 @@
+Classes in Java
+===============
+
+.. toctree::
+
+   background.rst
```

Comment:
note critical but you can leave the .rst off here i think!

Reply:
Done! Thanks!